codegen.yml

```diff
@@ -0,0 +1,6 @@
+overwrite: true
+schema: "schema.graphql"
+generates:
+  schema.d.ts:
+    plugins:
+      - "typescript"
```

Comment:
Is there way to add the configuration inline, or add it to `package.json`? I would prefer not to add new config files to the repository root.

Reply:
It seems not supported now.

Reply:
Wait for https://github.com/dotansimha/graphql-code-generator/pull/5144 to be released.